code/modules/mob/living/carbon/resist.dm

```diff
@@ -58,6 +58,9 @@
 
 
 /mob/living/proc/resist_grab()
+	var/mob/living/carbon/C = src
```

Comment:
You should probably be checking if they're a carbon before you cast them as one.

Reply:
Are you sure? Most people type check by doing that and the [byond reference](http://www.byond.com/docs/ref/#/proc/istype) says it’s ok so I’m not sure if there’s a dog difference

Reply:
@mattroks101 hey do you want me to change it to check if its a carbon before I cast it or is it good as is.

Reply:
@Garen7 
How about
```
/mob/living/carbon/resist_grab()
	return !handcuffed && ..()
```

Reply:
@LethalGhost 👍